Database/PetShareDbContext.cs

```diff
@@ -11,4 +11,21 @@ public sealed class PetShareDbContext : DbContext
 
     public DbSet<ShelterEntity> Shelters => Set<ShelterEntity>();
     public DbSet<PetEntity> Pets => Set<PetEntity>();
+    public DbSet<AnnouncementEntity> Announcements => Set<AnnouncementEntity>();
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
IDK if we need this method. Everyting should work fine without it, and it just repeats some things that EF Core would already do because of how we define properties in entity classes. Actually, this is the reason why we have `PetId` and `PetEntityId` columns: EF Core creates the latter by itself, and we explicitly tell it to create the former there.